core/pom.xml

```diff
@@ -306,6 +306,7 @@
                                 <include>org/elasticsearch/node/NodeMocksPlugin.class</include>
                                 <include>org/elasticsearch/node/MockNode.class</include>
                                 <include>org/elasticsearch/common/io/PathUtilsForTesting.class</include>
+                                <include>org/elasticsearch/rest/NoOpClient.class</include>
```

Comment:
In master I moved NoOpClient into the test framework so it could be shared by other tests. In 2.x there isn't a test framework to move it to so I just exported it in the test jar. That is kind of the test framework anyway.